kale/default_settings.py

```diff
@@ -59,3 +62,10 @@
 
 # CIPHER used by kale.crypt, must be 16-, 24-, or 36-byte string
 UTIL_CRYPT_CIPHER = '1234567890123456'
+
+# If Python 2 & Python 3 need to co-exist, set PICKLE_PROTOCOL=2
```

Comment:
Maybe say 'override PICKET_PROTOCOL to 2', otherwise, the comment is ambiguous: it may either as an instruction to the developer choosing to set the protocol, or just a general comment about what's happening in this setting.